e2e-tests/tfm/cypress/e2e/journeys/case-amendments/7. amendment-details-automatic-approval.spec.js

```diff
@@ -10,6 +10,10 @@ import {
 import { CURRENCY } from '../../../../../e2e-fixtures/constants.fixture';
 
 context('Amendments - automatic approval journey', () => {
+  const todayIsEndOfMonth = dateConstants.tomorrowDay === '01';
+  const todayIs28thFebruaryAndLeapYear = !todayIsEndOfMonth && dateConstants.todayDay === '28' && dateConstants.todayMonth === '02';
```

Comment:
I didn't test that this will fix the 28th Feb on a leap year issue but I think it will.
Could use [cy.clock](https://docs.cypress.io/api/commands/clock) to test this?

Reply:
can we take this out -- this is a bug. Could you raise a low priority ticket for it

Reply:
https://ukef-dtfs.atlassian.net/browse/DTFS2-7030